src/vs/workbench/contrib/terminal/browser/terminalView.ts

```diff
@@ -159,11 +176,66 @@ export class TerminalViewPane extends ViewPane {
 	public override getActionViewItem(action: Action): IActionViewItem | undefined {
 		if (action.id === TERMINAL_COMMAND_ID.SWITCH_TERMINAL) {
 			return this._instantiationService.createInstance(SwitchTerminalActionViewItem, action);
+		} else if (action.id === TERMINAL_COMMAND_ID.CREATE_PROFILE_BUTTON) {
+			if (this._tabButtons) {
+				this._tabButtons.dispose();
+			}
+			this._tabButtons = this._instantiationService.createInstance(CombinedButtonActionViewItem, this._getInitialTabActionBarArgs());
+			return this._tabButtons;
 		}
-
 		return super.getActionViewItem(action);
 	}
 
+	private async _updateTabActionBar(): Promise<void> {
+		const actions = await this._getTabActionBarArgs();
+		this._tabButtons?.update(actions);
+	}
+
+	private async _getTabActionBarArgs(): Promise<CombinedButtonArgs> {
+		const dropdownActions: IAction[] = [];
+		const submenuActions: IAction[] = [];
+
+		const profiles = await this._terminalService.getAvailableProfiles();
+		for (const p of profiles) {
+			dropdownActions.push(new MenuItemAction({ id: TERMINAL_COMMAND_ID.NEW_WITH_PROFILE, title: p.profileName, category: ContextMenuTabsGroup.Profile }, undefined, { arg: p, shouldForwardArgs: true }, this._contextKeyService, this._commandService));
+			submenuActions.push(new MenuItemAction({ id: TERMINAL_COMMAND_ID.SPLIT, title: p.profileName, category: ContextMenuTabsGroup.Profile }, undefined, { arg: p, shouldForwardArgs: true }, this._contextKeyService, this._commandService));
+		}
+
+		if (dropdownActions.length) {
+			dropdownActions.push(new SubmenuAction('split.profile', 'Split...', submenuActions));
+			dropdownActions.push(new Separator());
+		}
+
+		for (const [, configureActions] of this._dropdownMenu.getActions()) {
+			for (const action of configureActions) {
+				if ('alt' in action) {
+					dropdownActions.push(action);
+				}
+			}
+		}
+
+		const primaryAction = this._instantiationService.createInstance(MenuItemAction, { id: TERMINAL_COMMAND_ID.NEW, title: nls.localize('terminal.new', "New Terminal"), icon: Codicon.plus }, undefined, undefined);
+		const secondaryAction = this._instantiationService.createInstance(MenuItemAction, { id: 'more', title: 'more...', icon: Codicon.chevronDown }, undefined, undefined);
```

Comment:
The editor uses "More Actions...". We could also use "Launch Profile..." which is probably better even though there is some config stuff in there too.